### DISCUSSION TITLE
LoginForm.validate() - sanity check

### DISCUSSION THREAD

**Discussion by OldTinfoil** [pallets-eco/flask-security#725] [General]
Didn't want to post an issue in case I'm doing something wrong.

Using:
- wtforms=3.0.1
- Flask-WTF=1.1.0
- Flask-Security-Too=5.0.2

Using the default login page, I'm getting the following error when I attempt to login:
```
...
File "/path/to/project/venv/lib/python3.11/site-packages/flask_security/views.py", line 177, in login
    if form.validate_on_submit():
       ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/to/project/venv/lib/python3.11/site-packages/flask_wtf/form.py", line 86, in validate_on_submit
    return self.is_submitted() and self.validate(extra_validators)
```
the above `form` object is [LoginForm](https://github.com/Flask-Middleware/flask-security/blob/master/flask_security/forms.py#L459). Its validation function is not compatible with FlaskWTF's `validate_on_submit()` because it's not passing any (non) keyword arguments to unpack.

I have spent all day untangling my dependency hell in my project - but I'll revisit this tomorrow with a clean sandbox project to attempt to recreate the issue.

**Comment by jwag956** [comment]
Please see: https://github.com/wtforms/flask-wtf/issues/547
Pinning Flask-WTForms to 1.0.1 should get you going again.
